data/repos.yml

```diff
@@ -468,20 +588,44 @@
   team: "#data-products"
   type: Data science
 
+# Guessing, based on existing entry for `govuk-feedback-pipeline`?
+- repo_name: govuk-feedback-analysis
+  team: "#data-products"
+  type: Data science
```

Comment:
@nacnudus

Reply:
@j-marvin-GDS Should the govuk tag be removed, or should this be documented alongside other GOV.UK apps?

Reply:
Just nudging on this @j-marvin-GDS ?

Reply:
Hi, I'm not sure if the tag should be removed - how can I tell if a repo should have a GOVUK tag or not?

Thanks!

Reply:
Hi @j-marvin-GDS 👋 

Thanks for the question - I think there's a wider conversation to be had about how Data Products (and indeed other directorates like Pay and Notify) fit into GOV.UK. I'll start a conversation with GOV.UK Senior Tech about this.

In the meantime, my attempted answer to your question is this:

Some repos are owned by the Data team but GOV.UK (and particularly GOV.UK 2nd line) need to know about it. See [thoughts about govuk-knowledge-graph-search](https://github.com/alphagov/govuk-developer-docs/pull/3913/files#r1133680060) or [thoughts about govuk-s3-mirror](https://github.com/alphagov/govuk-developer-docs/pull/3913/files#r1132599575), which are both tightly coupled to some things maintained by GOV.UK. I think as a general rule, if your repo isn’t particularly tied to something that is maintained outside of the Data team, then it doesn’t need to be tagged as `govuk`.

I suspect `govuk-feedback-analysis` falls into the latter category and should just have its topic removed?

Reply:
Thanks Chris, I agree with this, happy to have the tag removed